src/graph/CMakeLists.txt

```diff
@@ -63,5 +63,5 @@ nebula_add_library(
     AdminJobExecutor.cpp
     LookupExecutor.cpp
 )
-nebula_add_subdirectory(test)
+# nebula_add_subdirectory(test)
```

Comment:
why remove this config?

Reply:
talked offline, this for fast build. while enable UT, will roll back in the end.